src/layouts/index.js

```diff
@@ -2,10 +2,13 @@
  * Copyright (c) 2013-present, Facebook, Inc.
  *
  * @emails react-core
+ * @flow
  */
 
 'use strict';
 
+// FIXME: I actually cannot figure out where this Template is used
```

Comment:
This is a built-in Gatsby thing: https://www.gatsbyjs.org/docs/building-with-components/#layout-components